inspire/base/templates/header.html

```diff
@@ -128,14 +104,37 @@
   </nav>
 </section>
 
-{% if request.path  == "/search" %}
+{% if request.path  == "/search" or request.path | is_collection_path %}
+
+  {% if request.path  == "/search" %}
+    {% set collection_name = request.args.get('cc') %}
+  {% else %}
+    {% set collection_name = request.path | check_collection %}
```

Comment:
`request.path` will return `collection/hep` for http://localhost:4000/collection/hep , however we only need the name of the collection: the `hep` part